core/src/main/java/org/apache/accumulo/core/data/impl/KeyExtent.java

```diff
@@ -383,32 +384,14 @@ public static Mutation getPrevRowUpdateMutation(KeyExtent ke) {
     return m;
   }
 
+  private static final Comparator<KeyExtent> COMPARATOR = Comparator
```

Comment:
This could use a better name (`KE_COMPARATOR` or `KEYEXTENT_COMPARATOR`?).

Reply:
I think this name is ok because its the comparator for the type in which its defined.

Reply:
Perhaps. It can just get confusing when looking at multiple classes, each of which have a `COMPARATOR` of the same name. At least for me, it causes me to step back and quadruple check which class I'm in and which comparator I was looking at. It would be easier for me, if the names were such that I could more easily bin them into separate locations in my brain.

Reply:
Changing the name makes looking at a single line in a diff easier, however I think it comes at the cost of adding redundant information to the source code.   Lots of redundant information like this negatively impacts the signal to noise ratio making reading the file unpleasant.

I think the name `KEYEXTENT_COMPARATOR` is redundant information because of the following :
 * The type is ` Comparator<KeyExtent>`
 * The variable is defined in `KeyExtent`
 * The variable is only used in `KeyExtent.compareTo`
 * If someone needed to add context for clarity they could write `KeyExtent.COMPARATOR`.   However I think that context is not needed in the current code.

Reply:
I think this comes down to a difference in the way our brains might be able to juggle and bin different bits of information in our heads. I agree it's redundant, but I think that redundancy helps me organize and read the code. The suggestion to write it as `KeyExtent.COMPARATOR` works, too, but... if it's not written that way already, it doesn't help the reader.

In any case, I don't feel strongly enough to hold up the PR. It's just a personal preference that helps my brain read the code. My personal preference need not hold up the PR. Do whatever you think best. :smiley_cat: